tests/test_ble_utils_service_targets.py

```
@@ -416,9 +433,12 @@ def test_management_helpers_cover_factory_and_target_edge_paths(
     monkeypatch: pytest.MonkeyPatch,
 ) -> None:
     """Management helper branches should handle factory fallback and target drift."""
+
     def _reject_kwarg(address: str, **kwargs: object) -> DummyClient:
         if kwargs:
-            raise TypeError("factory() got an unexpected keyword argument 'log_if_no_address'")
+            raise TypeError(
+                "factory() got an unexpected keyword argument 'log_if_no_address'"
+            )
```

Ruff (0.15.6)

[warning] Avoid specifying long messages outside the exception class

(TRY003)